src/EA.Iws.Web/Areas/AdminExportAssessment/Views/Overview/Sections/_TransitStates.cshtml

```diff
@@ -27,6 +29,8 @@
                     <div class="column-half">
                         @transitState.ExitPoint.Name,
                         @transitState.Country.Name
+                        
+                        <div class="change-answer">@Html.ActionLink("Change", "TransitExitPoint", "UpdateJourney", new { id = Model.NotificationId, transitStateId = transitState.Id }, null)</div>
```

Comment:
Need to add logic for hiding link for standard users.

Reply:
Fixed by this [commit](https://github.com/DEFRA/prsd-iws/pull/1005/commits/7e7f017a72489e3d5f2d0af680f726828fede152)